src/services/ui/src/pages/actions/setups/_.ts

```diff
@@ -1,7 +1,7 @@
-import { ZodObject } from "zod";
+import { ZodEffects, ZodObject } from "zod";
 import { AttachmentRecipe } from "@/lib";
 
 export type FormSetup = {
-  schema: ZodObject<any>;
+  schema: ZodObject<any> | ZodEffects<any>;
```

Comment:
Looks good. I was just curious what ZodEffects are. Is this for the super refine business

Reply:
Yes, ZodEffects is the return type of `superRefine` so I plugged it in as a union here :)